astropy/io/ascii/tests/test_html.py

```diff
@@ -16,7 +16,7 @@
 
 from .common import setup_function, teardown_function
 from ....tests.helper import pytest
-from ....extern.six.moves import cStringIO
+from ....extern.six.moves import range, cStringIO as StringIO
```

Comment:
Why is there a need to rename `cStringIO`? Won't that just be adding to the confusion?